app-accessibility/flite/flite-2.1.recipe

```diff
@@ -0,0 +1,142 @@
+SUMMARY="A light-weight text-to-speech engine"
+DESCRIPTION="Flite is a text-to-speech engine developed by the Carnegie \
+Mellon University. As a light-weight alternative to the Festival engine, it \
+is desinged to fit into small embedded machines as well as large-scale \
+servers. In addition to its native format flite supports voices generated by \
+the FestVox project as well."
+HOMEPAGE="http://www.festvox.org/flite/
+	https://github.com/festvox/flite"
+COPYRIGHT="1986 University of Toronto
+	1992, 1995 Markus Mummert
+	1996-2004 University of Edinburgh, UK
+	1999-2014 Language Technologies Institute, Carnegie Mellon University
+	2001 Cepstral, LLC
+	2001 Sun Microsystems, Inc.
+	2003-2004 Nagoya Institute of Technology
+	2003-2004 Nara Institute of Science and Technology
+	2008-2012 University of Illinois at Urbana-Champaign"
+# Although individually many files have different licenses, as a whole this
+# license covers all the files.
+LICENSE="Flite"
+REVISION="1"
+SOURCE_URI="http://www.festvox.org/${portName}/packed/${portName}-${portVersion}/${portName}-${portVersion}-release.tar.bz2"
```

Comment:
Using $portName here wont work for secondaryArchSuffix, it will change the name to "flite_x86"